lib/karafka/consumers/responders.rb

```diff
@@ -11,7 +11,7 @@ module Responders
       def respond_with(*data)
         Karafka.monitor.notice(self.class, data: data)
         # @note we build a new instance of responder each time, as a long running (persisted)
```

Comment:
we build a new instance of responder each time, as a long-running (persisted) consumers can respond multiple times during the lifecycle